source/builds.md

```diff
@@ -112,4 +112,4 @@ Unless required by applicable law or agreed to in writing, software distributed
 "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations under the License.
 
-Next: [Release Guidelines](releases.html)
+Next: [Release Guidelines](release-guidelines)
```

Comment:
`.html`?

Reply:
intentionally not adding extension.

Maybe one day /release-guidelines will be a folder, or `.jsp`.  
Would you accept such approach?

Reply:
Why folder? And not `.jsp` for sure :)

Reply:
... the point was missed. Just adding `.html`